.env.test

```
@@ -0,0 +1,36 @@
+VITE_APP_API_URI=https://kultus.api.test.hel.ninja/graphql
+VITE_APP_API_REPORT_URI=https://kultus.api.test.hel.ninja/reports
+VITE_APP_LINKEDEVENTS_API_URI=https://linkedevents.api.test.hel.ninja/v1
+
+# OIDC client configuration
+VITE_APP_OIDC_AUDIENCES=kultus-api-test,profile-api-test
+VITE_APP_OIDC_RETURN_TYPE=code
+VITE_APP_OIDC_SERVER_TYPE=KEYCLOAK
+VITE_APP_OIDC_API_CLIENT_ID=kultus-api-test
+VITE_APP_OIDC_AUTHORITY=https://tunnistus.test.hel.ninja/auth/realms/helsinki-tunnistus
+VITE_APP_OIDC_CLIENT_ID=kultus-admin-ui-test
+VITE_APP_OIDC_SCOPE=openid profile email
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_APP_OIDC_SCOPE key should go before the VITE_APP_OIDC_SERVER_TYPE key

(UnorderedKey)